post_deployment_tests/deployment_test.py

```diff
@@ -0,0 +1,338 @@
+"""module used for testing a deployed archive interface."""
+
+import os
+import requests
+import unittest
+
+# url for the archive interface just deployed.
+# Clean Local files will remove all the test file generated from where the script is run if true
+# Clean archive files will remove the files from the archive, assuming that this process can access the
+# path for the archive files
+# archive prefix is only used if cleaning archive files generated by this test script
+ARCHIVEURL = 'http://127.0.0.1:8080/'
+CLEANLOCALFILES = True
+CLEANARCHIVEFILES = True
+ARCHIVEPREFIX = '/home/knig880/archive/'
```

Comment:
Try to do something similar with environment variables in the main system. Please replace with `getenv` calls.